core/src/test/java/org/elasticsearch/action/admin/cluster/snapshots/status/SnapshotStatusTests.java

```diff
@@ -0,0 +1,89 @@
+package org.elasticsearch.action.admin.cluster.snapshots.status;
```

Comment:
The CI tests just complained about missing license headers for this new test file. Can you please copy over the header comments from any existing test to this one? Sorry I missed this during review.